pkg/policyreport/policyreport.go

```diff
@@ -104,6 +104,11 @@ func updateResults(oldReport, newReport map[string]interface{}, aggregatedReques
 		return nil, hasDuplicate, err
 	}
 
+	if summaryResults == nil {
+		fmt.Println("Debug: summaryResults is nil, expected array. Results from old report: ", results)
```

Comment:
Can we use a logger here? That way we can set the severity to info. Not sure if we should always print this with `println`.